src/ck-ldb/CentralLB.C

```diff
@@ -673,7 +658,7 @@ void CentralLB::LoadBalance()
   for (proc = 0; proc < clients; proc++) statsMsgsList[proc] = NULL;
 #endif
 
-  theLbdb->ResetAdaptive();
+  lbmgr->ResetAdaptive();
```

Comment:
Isn't adaptiveLB something that was removed in PR 1? or is this something else unrelated?

Reply:
No, it's for Metabalancer.